README.md

```diff
@@ -1,2 +1,28 @@
 # Portfolio-Website
 Personal Portfolio website as a microverse day3 project
+
+># Project Details
+>this project focuses on building a personal portfolio website
+which will showcase the projects you have worked on as a developer, your personal achievements so far and why you think you fits in as a software developer in the tech industry.
+this is a complete website that microverse students are undertaken to enhance thier coding skills. 
+
+
+# Installation process(Packages)
+first add the .github folder followed by the steps below
+*npm install --save-dev hint@7.x
+*Copy .hintrc to the root directory of your project.
+*run npx hint .
+*npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+# Languages used
+*Html
+*Css
+
+👤 **Author**
+Sagoe Bright Marcus
+
+# Social Media Links
+
+- GitHub: [@github](https://www.linkedin.com/in/bright-marcus-sagoe-74335820a/)
+- Twitter: [@facebook](https://web.facebook.com/?_rdc=1&_rdr)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [OPTIONAL] Great job using the Microverse Template here 💯👌 and I think it would be better to add the MIT.md file, you can use [this template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) and don't forget to customize it 👌.

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs.